ReadMe.md

```diff
@@ -31,16 +31,19 @@ You can also install Vagrant/Virtualbox on Windows by running `choco install pac
 ### Preparing the Testing Environment
 
  * Ensure setup above is good on your machine.
- * Open a command line (`PowerShell.exe`/`cmd.exe` on Windows, `bash` everywhere else) and navigate to this directory of the repository. You know you are in the right place when you do a `dir` or `ls` and `Vagrantfile` is in your path.
+ * Fork and Clone this repository
+ * Open a command line (`PowerShell.exe`/`cmd.exe` on Windows, `bash` everywhere else) and navigate to the root folder of the repository.  You know you are in the right place when you do a `dir` or `ls` and `Vagrantfile` is in your path.
    * No idea if bash on Windows (through Git/CygWin) is supported. If you run into issues, it is better to just use `PowerShell.exe` or `cmd.exe`. Please do not file issues stating it doesn't work.
- * Run `vagrant up` to prepare the machine for testing. Note that it will take quite awhile the first time you need to download the [box from Atlas](https://atlas.hashicorp.com/ferventcoder/boxes/win2012r2-x64-nocm). Once it has downloaded it will import the box and apply the scripts and configurations to the box as listed inside the `Vagrantfile`.
- * Now the box is ready for you to start testing against. Run `vagrant sandbox on`.
+ * Run `vagrant up` to prepare the machine for testing. 
+   * **Note** due to the way that vagrant works, the first time that you run this command, the vagrant box named __ferventcoder/win2012r2-x64-nocm__ needs to be downloaded from the [Atlas website](https://atlas.hashicorp.com/ferventcoder/boxes/win2012r2-x64-nocm).  This will take quite a while, and should only be attempted on a reasonably fast connection, that doesn't have any download limit restrictions. Once it has downloaded it will import the box and apply the scripts and configurations to the box as listed inside the `Vagrantfile`.  You can find the downloaded box in the `~/.vagrant.d` or `c:\users\username\.vagrant.d` folder.
+ * Now the box is ready for you to start testing against. 
+ * Using the [vagrant plugin](https://github.com/jedi4ever/sahara) that was installed earlier, it is now possible to take a snapshot of the VM, so that after testing packages, the VM can be returned to this known "good" state.  This snapshot can be taken using the following command: `vagrant sandbox on`.

```

Comment:
This is informational. However there still needs to be an imperative. 

Run `vagrant sandbox on`.